examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h

```diff
@@ -1,18 +1,14 @@
-
-
 #ifndef OP_APP_MSGIDS_H
 #define OP_APP_MSGIDS_H
 
-#define OP_FLAG_MID 		0x1800
-#define OP_CMD_MID  		0x1801                  
-#define OP_PERF_ID   		0x1802            
-#define OP_TICK_MID   		0x1803              
-#define OP_HK_MID     		0x1804     
-#define OP_PROPSTATE_MID 	0x1805
-
-#define ECI_TBL_MANAGE_MID 	0x1806
-#define ECI_SEND_HK_MID 	0x1807           
+#define OP_FLAG_MID         0x1AA0
+#define OP_CMD_MID          0x1AA1
+#define OP_PERF_ID          0x1AA2
```

Comment:
It actually appears that this is not a MID, its a performance ID, so perhaps should be numbered separately from the sequence of APID's? Might be worth it to look for any standard that the CFE might follow for numbering these just to set a good example.

Reply:
Folded this into the new commits. I was more regimented about cmd/tlm designations.